### ISSUE TITLE
App example using webpack: Make it more interactive

### ISSUE THREAD

**Issue by rxaviers** [jquery/globalize#489]
Allow user to change number, currency, and the number of remaining tasks for the pluralization example.

This issue has been motivated by @sompylasar comment:
> Maybe for the demo purposes also increment the `count` from `0` to some number and back to `0`? This would demonstrate how plurals work with different values.

**Comment by sotojuan** [comment]
Been working on this, it's not super pretty but it works.

![](http://i.imgur.com/17MnWYQ.png)

I'm taking a look at the style guide now, but if this looks good I can submit the PR.

**Comment by sotojuan** [comment]
Update: I moved the inputs to the table and added relative time and date format:

![](http://i.imgur.com/YuA1u0I.png)

Let me know how this looks!

**Comment by rxaviers** [comment]
@sotojuan this is looking great. Although, note the following.

When we change the formatter value only (e.g., changing the number in the first example), we use the same formatter to format various values, which is great. When we change the formatter options (e.g., changing the currency code, date pattern, or relative time unit), we are actually using different formatters, not reusing a formatter, which has its consequences: while it works just fine in development mode, where you can generate any formatter, it won't work in production mode, where it's expected that the formatter be compiled and available.

To make long story short, make dynamic the formatter values only, i.e.:
- number: it's ok.
- currency: the currency value 69900 (not the currency code).
- date: the date (not the format).
- relative time: the number of weeks (not the unit).

Does it sound good?

Thanks so far!

PS: Note you can make the relative time or patterns dynamic, but that would require a code analogous to https://github.com/jquery/globalize/pull/481#issuecomment-140340869.

**Comment by sotojuan** [comment]
Gotcha, I made the following changes:

* Currency input now changes the value
* Date input changes the date
* Relative time increases the seconds by the given input. Can enter negative numbers to decrease.

![](http://i.imgur.com/kmep4pl.png)

Let me know how this looks! I just need to handle bad inputs, add some comments, and look at the style guide again.

**Comment by rxaviers** [comment]
@sotojuan sorry for my delayed answer, it looks better. From the picture above, I read `Change format`, but I guess it's only a matter of updating the button title to `Change date` right?

**Comment by sotojuan** [comment]
Yep, thanks for catching that. I'm a bit busy now but later I will clean it up and follow the style guide and submit a PR. Thanks!

**Comment by rxaviers** [comment]
Excellent. Looking forward to it and thanks.
